src/TwigComponent/src/CVA.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent;
+
+/**
+ * @author Mathéo Daninos <matheo.daninos@gmail.com>
+ *
+ * CVA (class variant authority), is a concept from the js world.
+ * https://cva.style/docs
+ * The UI library shadcn is build on top of this principle
+ * https://ui.shadcn.com
+ * The concept behind CVA is to let you build component with a lot of different variations called recipes.
+ *
+ * @experimental
+ */
+final class CVA
+{
+    /**
+     * @var string|null
```

Comment:
```suggestion
     * @var string|list<string|null>|null
```

BTW, I'm suggesting this to help with https://github.com/symfony/ux/pull/1416#discussion_r1506202909